Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -199,6 +199,8 @@
     <string name="stereoscopy_convergence_description">Control the distance of the convergence plane, this is the distance at which objects will appear to be in front of the screen.\nA higher value creates stronger out-of-screen effects while a lower value is more comfortable.</string>
     <string name="stereoscopy_swap_eyes">Swap Eyes</string>
     <string name="stereoscopy_swap_eyes_description">Swap the left and right eye, mostly useful if you want to view side-by-side cross-eyed.</string>
+    <string name="backend_multithreading">Backend Multithreading</string>
+    <string name="backend_multithreading_description">Enables GPU multithreading. May slow emulation. If unsure leave unchecked(Vulkan only)</string>
```

Comment:
The part about Vulkan only would fit better at "enables GPU multithreading" than "if unsure leave unchecked".

Also, the multithreading is actually on the CPU, not on the GPU.

Reply:
Maybe "may affect performance" instead of "may slow emulation" as well? Since it does give speedup on some hardware.

Reply:
Fixed